.ci/azure/linux.yml

```diff
@@ -89,17 +89,17 @@ jobs:
     CCACHE_DIR: $(SHARE_DIR)/ccache/master/linux
 
   steps:
-  - script: |
-      curl -H Metadata:true --noproxy "*" "http://169.254.169.254/metadata/instance?api-version=2019-06-01"
-      whoami
-      uname -a
-      echo Python3 info ; which python3 ; python3 --version
-      echo Python info ; which python ; python --version
-      echo Java info ; which java ; java -version
-      echo gcc info ; which gcc ; gcc --version
-      echo cmake info ; which cmake ; cmake --version
-      lsb_release
-      env
+    - script: |
+        curl -H Metadata:true --noproxy "*" "http://169.254.169.254/metadata/instance?api-version=2019-06-01"
+        whoami
+        uname -a
+        echo Python3 info ; which python3 ; python3 --version
+        echo Python info ; which python ; python --version
+        echo Java info ; which java ; java -version
+        echo gcc info ; which gcc ; gcc --version
+        echo cmake info ; which cmake ; cmake --version
+        lsb_release
+        env
```

Comment:
Not sure if we need to change this. please revert since not neccassary.

Reply:
Of course, removed